### DISCUSSION TITLE
icon color change

### DISCUSSION THREAD

**Discussion by charith0418** [twbs/bootstrap#39614] [Q&A]
i want to know how to change color in icon , i used css styles but still not change, anyone can help me

**Comment by Alex-Toucan** [comment]
Hello. Have you tried using any of the text color classes? They can be found at https://getbootstrap.com/docs/5.3/utilities/colors/#colors. If you only have icons, just use the `color` CSS attribute on your icons.

Also, what method of using icons are you using, SVG tags or the I (italic) tags?

**Comment by abdullah26tr** [comment]
Hi!
If you're working with an icon that is an image file (like a PNG or SVG), you can use CSS to change its color.

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <style>
    .icon {
      fill: red; /* Change this to your desired color */
    }
  </style>
</head>
<body>
  <svg class="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-1 15h2v-2h-2zm0-4h2V7h-2z"/>
  </svg>
</body>
</html>
```

Copy and paste this code into an HTML file. You can change the fill property in the CSS to the color you desire.
